app/pages/health/ready/getReadyStatus.js

```diff
@@ -2,8 +2,20 @@ import { ApiProvider } from '../../../apiProvider';
 import { status } from '../status';
 
 export async function getReadyStatus() {
-  const buyingCatalogueApi = (await new ApiProvider().getBuyingCatalogueApiHealth()).data;
-  const documentApi = (await new ApiProvider().getDocumentApiHealth()).data;
+  let buyingCatalogueApi;
+  let documentApi;
+  // TODO: Global fix for unavailable endpoints.
```

Comment:
what is the global fix for unavailable endpoints?

Reply:
I'll remove the comments, but basically we don't handle the endpoints being unavailable anywhere in the app